cfe_internal/enterprise/federation/federation.cf

```diff
@@ -195,16 +195,17 @@ bundle agent transport_user
       edit_line => default:insert_lines("IdentityFile $(ssh_priv_key)");
 
   commands:
+    selinux_enabled.missing_ssh_context::
```

Comment:
`missing_ssh_context` is based on contexts of the **files** in the directory, not on the context of the directory itself, AFAICT.